grafana/dashboard.json

```diff
@@ -1,927 +1,921 @@
 {
-  "__inputs": [
-    {
-      "name": "DS_PROMETHEUS",
-      "label": "Prometheus",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "prometheus",
-      "pluginName": "Prometheus"
-    }
-  ],
-  "__requires": [
-    {
-      "type": "panel",
-      "id": "gauge",
-      "name": "Gauge",
-      "version": ""
-    },
-    {
-      "type": "grafana",
-      "id": "grafana",
-      "name": "Grafana",
-      "version": "7.2.2"
-    },
-    {
-      "type": "panel",
-      "id": "grafana-piechart-panel",
-      "name": "Pie Chart",
-      "version": "1.6.1"
-    },
-    {
-      "type": "panel",
-      "id": "graph",
-      "name": "Graph",
-      "version": ""
-    },
-    {
-      "type": "datasource",
-      "id": "prometheus",
-      "name": "Prometheus",
-      "version": "1.0.0"
-    },
-    {
-      "type": "panel",
-      "id": "stat",
-      "name": "Stat",
-      "version": ""
-    }
-  ],
-  "annotations": {
-    "list": [
+  "annotations":{
+    "list":[
       {
-        "builtIn": 1,
-        "datasource": "-- Grafana --",
-        "enable": true,
-        "hide": true,
-        "iconColor": "rgba(0, 211, 255, 1)",
-        "name": "Annotations & Alerts",
-        "type": "dashboard"
+        "builtIn":1,
+        "datasource":{
+          "type":"datasource",
+          "uid":"grafana"
+        },
+        "enable":true,
+        "hide":true,
+        "iconColor":"rgba(0, 211, 255, 1)",
+        "name":"Annotations & Alerts",
+        "target":{
+          "limit":100,
+          "matchAny":false,
+          "tags":[
+
+          ],
+          "type":"dashboard"
+        },
+        "type":"dashboard"
       }
     ]
   },
-  "editable": true,
-  "gnetId": null,
-  "graphTooltip": 0,
-  "id": null,
-  "iteration": 1603732792335,
-  "links": [],
-  "panels": [
+  "editable":true,
+  "fiscalYearStartMonth":0,
+  "graphTooltip":0,
+  "iteration":1651440933266,
+  "links":[
+
+  ],
+  "liveNow":false,
+  "panels":[
     {
-      "datasource": "${DS_PROMETHEUS}",
-      "fieldConfig": {
-        "defaults": {
-          "custom": {},
-          "mappings": [
-            {
-              "from": "",
-              "id": 0,
-              "text": "Online",
-              "to": "",
-              "type": 1,
-              "value": "1"
-            },
+      "datasource":{
+        "type":"prometheus",
+        "uid":"PBFA97CFB590B2093"
```

Comment:
This only exists in your installation and can't be reused

Reply:
I've added a data source selection, let me know what you think!